mock-api/snapshot.ts

```diff
@@ -77,7 +77,7 @@ export const snapshots: Json<Snapshot>[] = [
     state: 'ready',
   },
   {
-    id: '7fc6ca11-452e-d3e4-9e1c-752ff615abea',
+    id: '75832119-7d9b-4e91-a7c6-76b996b69aaa',
```

Comment:
Turns out this was an invalid UUID, correctly rejected by Zod because of the d in d3e4, which is supposed to be a 4 for UUID v4. Solved this on the last attempt, see https://github.com/oxidecomputer/console/pull/2828#discussion_r2096366571.